ci-operator/step-registry/ovn/sdn-migration/feature-migration/config-egress-firewall/ovn-sdn-migration-feature-migration-config-egress-firewall-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: ovn-sdn-migration-feature-migration-config-egress-firewall
+  from_image:
+    name: cli-jq
+    namespace: ocp
+    tag: latest
+  commands: ovn-sdn-migration-feature-migration-config-egress-firewall-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: EGRESS_FIREWALL_SPEC
+    default: '{"egress":[{"to":{"dnsName":"docs.openshift.com"},"type":"Allow"},{"to":{"cidrSelector":"8.8.8.8/32"},"type":"Allow"},{"to":{"cidrSelector":"0.0.0.0/0"},"type":"Deny"}]}'
```

Comment:
Why do we need to provide allow access to docs.openshift.com? Same question goes for the validation component of the test.

Reply:
It's just a randomly picked URL for testing, no particular reason.